src/user/MOM_wave_interface.F90

```diff
@@ -322,6 +323,9 @@ subroutine MOM_wave_interface_init(time, G, GV, US, param_file, CS, diag )
          "STK_BAND_COUPLER is the number of Stokes drift bands in the coupler. "// &
          "This has to be consistent with the number of Stokes drift bands in WW3, "//&
          "or the model will fail.",units='', default=1)
+      call get_param(param_file, mdl, "SURFBAND_WAVENUMBERS", CS%WaveNum_Cen, &
+           "Central wavenumbers for surface Stokes drift bands.", &
+           units='rad/m', default=0.12566, scale=US%Z_to_m)
       allocate( CS%WaveNum_Cen(CS%NumBands) )
```

Comment:
The allocate call for CS%WaveNum_Cen needs to occur before it can be read into with the get_param call.  I have an update to this PR that fixes this problem, but I can not push it back to github at the moment because Gaea is reporting network connectivity issues.

Reply:
Gaea can not connect to the internet from interactive nodes, and I was foolishly pushing from the wrong window.  The updated version of this PR addresses this issue.